app/retail/views.py

```diff
@@ -344,6 +344,19 @@ def mission(request):
     return TemplateResponse(request, 'mission.html', context)
 
 
+def vision(request):
+    """Render the Vision response."""
+    context = {
+        'is_outside': True,
+        'active': 'vision',
+        'title': 'Vision',
+        'card_title': _('Gitcoin is a mission-driven organization.'),
+        'card_desc': _('Our mission is to grow open source.'),
+        'avatar_url': static('v2/images/grow_open_source.png'),
```

Comment:
`Gitcoin's Vision for a web3 world is a positive vision of a future in which it's easy for developers to find paid work in open source.'